src/tls/openssl/tls.c

```diff
@@ -142,6 +143,7 @@ int tls_alloc(struct tls **tlsp, enum tls_method method, const char *keyfile,
 	if (!tls)
 		return ENOMEM;
 
+	tls->sverify = true;
```

Comment:
i think `tls->server_verify` is easier to read and remember

Reply:
more consistent `verify_server`

Reply:
ok.